Composer/packages/server/src/controllers/storage.ts

```diff
@@ -16,8 +16,8 @@ function createStorageConnection(req: Request, res: Response) {
 }
 
 function updateCurrentPath(req: Request, res: Response) {
-  StorageService.updateCurrentPath(req.body.path);
-  res.status(200).json('success');
+  const storages = StorageService.updateCurrentPath(req.body.path);
```

Comment:
Prefer not to keep one-off result in a constant. You can just pass it in.

But depends on the team or other code, they may like this one.